addons/mail/static/src/models/chatter.js

```diff
@@ -229,6 +229,13 @@ registerModel({
             }
             return clear();
         },
+        /**
+         * @private
+         * @returns {boolean}
+         */
+        _computeHasReadAccess() {
+            return this.thread && !this.thread.isTemporary && this.thread.hasReadAccess;
```

Comment:
`return Boolean(this.thread && !this.thread.isTemporary && this.thread.hasReadAccess)`